standalone/standalone.tpl

```diff
@@ -2191,8 +2209,10 @@ def train(
     epoch_num = None
     if training_phase == "1":
         epoch_num = training_1_epoch_num
+        effective_batch_size = training_1_effective_batch_size
```

Comment:
I think setting it to "None" here would just make it "None" always and would break. 

Do you mean at line 895 instead?
https://github.com/opendatahub-io/ilab-on-ocp/blob/fcc47a3a425f025ee4fe7aeba018f7f82136f8d5/standalone/standalone.tpl#L895-L897

Reply:
nvm. I see what you mean